### DISCUSSION TITLE
The survey link in codespace's select form is broken

### DISCUSSION THREAD

**Discussion by Anarbb** [community/community#56556] [Codespaces]
### Select Topic Area

Bug

### Select Feature Area

- [X] Issues
- [ ] Projects

### Body

Hello,

Not sure if this is the right place for this, but the link that leads to the survey to contact the team to get access to 32-Core CPUs and GPU codespaces is broken [https://github.co/codespaces_sku_request](https://github.co/codespaces_sku_request)
Here is the image for more clarification (it's the "Contact our team link"):
![image](https://github.com/community/community/assets/65725801/ba2a4892-239d-4bbe-8866-959fb560f1b8)

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Anarbb ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Comment by joshaber** [comment]
Thanks @Anarbb! We're working on getting this updated 👍
